lib/components/Meta/Meta.spec.jsx

```diff
@@ -1,9 +1,7 @@
 import React from 'react';
-import { mount, shallow } from 'enzyme';
+import { shallow } from 'enzyme';
```

Comment:
This is not new to this PR and in fact I think what I did back in the day but perhaps we should not shallow on components that wrap other components because the end snapshot would not be what the user sees. This is one example of a current snapshot
`<span
  className="root variantPrimary"
>
  <Component
    className="icon"
  />
  <Text
    className="withIcon"
  >
    Hello World!
  </Text>
</span>`

This may or may not be desirable as the wrapped component is tested separately but my vote would go to render for wrapping components